tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py

```diff
@@ -506,9 +506,9 @@
     ),
     esp_number_of_stages=2,
     esp_reference_density=Scalar(1000.0, "kg/m3"),
-    user_defined_esp_table=TABLE_PUMP_DESCRIPTION,
```

Comment:
Why did you remove `user_defined_esp_table` and `esp_parameters`?

Reply:
Sorry, this was a mistake.

Reply:
As we realized later: the strange thing was that it was removed from this file, but not from the schema; that as been fixed since then.